book/python/api.md

```diff
@@ -444,7 +444,7 @@ A complete `KafkaSink` encoder example that takes a word and sends it to the par
 @wallaroo.encoder
 def kafka_encode(data):
     word = data[:]
-    letter_key = data[0]
+    letter_key = data[0:1]
```

Comment:
I wonder if it would help to include a comment on how `data[0:1]` is different than `data[0]` between python2 and 3

Reply:
Sure.